### DISCUSSION TITLE
process is not defined

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#9927] [CLI - Electron mode]
I've recently created a new project in Quasar and I need to use fs.existsSync and some other path related file operations like the following, for an eletron desktop app. 

```
const content = fs.readFileSync(filepath, 'utf8');
const loc = path.join(path.dirname(filepath), 'sample_presets');
```

How can I get 'path' and 'fs' working? They worked in older projects i made over a year ago.

however i get the following errors after running **quasar add path**. did they remove the options from being usable? 

```

Uncaught ReferenceError: process is not defined
    at eval (path.js?1ff0:25)
    at Object../node_modules/path/path.js (vendor.js:2658)
    at __webpack_require__ (app.js:272)
    at fn (app.js:536)
    at eval (utils.js?025e:2)
    at Module../src/utils.js (app.js:184)
    at __webpack_require__ (app.js:272)
    at fn (app.js:536)
    at eval (actions.js:11)
    at Module../src/store/app/actions.js
```

**Comment by ap1969** [comment]
Hi @JokerMartini  - Are you running Quasar v2?

I've just upgraded an app from v1 to v2, and process is now undefined: https://github.com/quasarframework/quasar/discussions/10088

**Reply by JokerMartini** [reply]
Yeah I am

**Reply by ap1969** [reply]
I just resolved the issue, but I'm not sure how. See the last post in my thread.
